src/ripple/rpc/handlers/LedgerData.cpp

```diff
@@ -50,11 +50,12 @@ Json::Value doLedgerData (RPC::Context& context)
     if (!lpLedger)
         return jvResult;
 
-    uint256 resumePoint;
+    boost::optional<ReadView::key_type> key;
+    key.emplace();
```

Comment:
I need the `boost::optional` later as a return from `succ` - this means I only need the one variable.